lms/static/sass/discussion/views/_thread.scss

```diff
@@ -34,6 +34,7 @@ body.discussion, .discussion-module {
     }
 
     .posted-details {
+      @extend %t-copy-sub2;  // Edraak: Corrected the font-size
```

Comment:
Too large, no? The production still looks gentler.

**Beta**
<img width="824" alt="screen shot 2016-06-28 at 10 42 47 am" src="https://cloud.githubusercontent.com/assets/11036472/16407573/17924c86-3d1d-11e6-9b2f-80a0db98a8c8.png">

**Production**
<img width="805" alt="screen shot 2016-06-28 at 10 42 40 am" src="https://cloud.githubusercontent.com/assets/11036472/16407576/1a42bb1e-3d1d-11e6-9957-306a5935a880.png">